about/index.php

```diff
@@ -135,18 +135,31 @@
   In addition to my software work, I also enjoy swimming and other water sports, baking and cooking, day hiking, and
   working with youth. Since 2015, my wife and I have been welcoming foster children into our home.
 </p>
+  <p style="margin-top: 32px" class="center"><br /></p>
+  <h4 class="caption">DG</h4>
+  <br />
+  <p class="people-text">
+    I'm from Northern Ireland and I have a Natural Sciences degree from Cambridge.
+    I've been working on the Linux side of Keyman since May 2018. Prior to that I've
+    been working as a software developer and/or volunteering on various free and
+    open software projects.
+  </p>
+  <p class="people-text">
+    I enjoy audax, other non-speedy cycling, and board games.
+  </p>
+
+  <p style="margin-top: 32px;" class="center"><img class="inline" src="<?php echo cdn("img/eberhard.jpg"); ?>" /><br /><br /></p>
```

Comment:
You've attached a png instead of jpg to the PR. Otherwise, lgtm

Reply:
Thanks for noticing!